packages/mui-system/src/Container/createContainer.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { Interpolation, StyledComponent } from '@mui/styled-engine';
+import { Interpolation, MUIStyledComponent as StyledComponent } from '@mui/styled-engine';
```

Comment:
This change is causing my build system to fail.
```
node_modules/@mui/system/Container/createContainer.d.ts:2:25 - error TS2724: '"@mui/styled-engine"' has no exported member named 'MUIStyledComponent'. Did you mean 'StyledComponent'?

2 import { Interpolation, MUIStyledComponent as StyledComponent } from '@mui/styled-engine';
```

Reply:
What are the versions of the mui packages that you've installed? Mainly `@mui/system` and `@mui/styled-engine` or `@mui/styled-engine-sc`.

Reply:
You can explicitly pin the package version of `@mui/styled-engine` to `5.14.14` till we do the next release.